backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java

```diff
@@ -121,7 +120,7 @@ private void deleteProjects(Session session) {
         projectIds.add(projectEntity.getId());
       }
       try {
-        deleteRoleBindingsForProjects(projectEntities);
+        roleService.deleteProjectResources(projectIds);
       } catch (StatusRuntimeException ex) {
```

Comment:
We need to ignore just chosen exceptions instead of all.

Reply:
https://vertaai.slack.com/archives/CJG0NL86B/p1607920510000500?thread_ts=1607681257.124500&cid=CJG0NL86B

Reply:
So this code is fine here.

Reply:
The link says that UAC will return 200 in case of success. This means that we should _stop_ in case of an error, instead of going ahead and deleting things from the db. Once we delete from the db, we'll never call this deletion again, which will cause a leak.

Reply:
`roleService.deleteProjectResources(projectIds);` bulk deletion call to UAC if we stop in case error then all the projects are skipped for deletion due to the one project. so do you think we call UAC for each project instead of bulk??

Reply:
What we cannot have it fail and then we proceed like it didn't fail. That is clearly a bug that will leave to inconsistent behavior. Please check the other similar pieces of code for this same bug and let's fix them in another PR.

Reply:
Started a thread for this on slack https://vertaai.slack.com/archives/CJG0NL86B/p1608020346023100
No need to create a new PR for this. it is not a complex logic. it just needs to move `roleService.deleteProjectResources(projectIds);` in the same try-catch to stop going for the further process.